<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_Read">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Read image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cam_L">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>30</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CLOSE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cam_R">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pause">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_play">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="save_L">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>0</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="save_R">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>460</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_disp">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>510</y>
      <width>501</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string>Disparity</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_pseudo_color">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pseudo-color</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_do_depth">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>451</y>
      <width>73</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Do depth</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_color_table">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>920</y>
      <width>300</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>ColorTable</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_disp_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>550</y>
      <width>501</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string>Disparity</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
